classes/controller/class-ensapi-controller.php

```diff
@@ -302,30 +310,33 @@ public function get_supporter_questions() {
 	 */
 	public function get_supporter_question_by_id( $question_id ) {
 		$response['body'] = get_transient( 'ens_supporter_question_by_id_response_' . $question_id );
-		if ( ! $response['body'] ) {
-			$url = self::ENS_SUPPORTER_URL . '/questions/' . $question_id;
+		if ( $response['body'] ) {
```

Comment:
We can do the same as here https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/308/files#r449458709